crates/iota-sdk/src/wallet_context.rs

```diff
@@ -55,6 +55,24 @@ impl WalletContext {
             )
         })?;
 
+        if let Some(active_address) = &config.active_address {
+            let addresses = match &config.keystore {
+                Keystore::File(file) => file.addresses(),
+                Keystore::InMem(mem) => mem.addresses(),
+            };
+            ensure!(
+                addresses.contains(active_address),
+                "active address does not belong to configured keystore"
+            );
+        }
+
+        if let Some(active_env) = &config.active_env {
+            ensure!(
+                config.get_env(active_env).is_some(),
+                "unknown active environment"
```

Comment:
Wondering if we should add the path to the config folder in the message or give some advice how this can be resolved

Reply:
yeah, why not.. I'll try to come up with something reasonable.